content/influxdb/v1.8/tools/influx_inspect.md

```diff
@@ -452,6 +454,80 @@ The flag to report exact cardinality counts instead of estimates.
 Default value is `false`.
 Note: This can use a lot of memory.
 
+### `report-disk`
+
+Use the `report-disk` command to review disk usage by shards and measurements for TSM files in a specified directory. Useful for determining disk usage for capacity planning and identifying which measurements or shards are using the most space.
+
+Calculates the total disk size in bytes by database (`db`), retention policy (`rp`), shard (`shard`), tsm file (`tsm_file`), and measurement (`measurement`).
```

Comment:
More accurate would be 'or measurement':

```
Calculates the total disk size in bytes by database (`db`), retention policy (`rp`), shard (`shard`), tsm file (`tsm_file`), or by measurement (`measurement`).
```

Reply:
But the above is a bit confusing, I'm sure it can be worded better.

Reply:
Thank you, @lesam, please verify the changes in the latest commit are accurate.